src/services/RetroGenerationService.ts

```diff
@@ -28,6 +28,12 @@ class RetroGenerationService {
             // Fetch and populate the Retrokey template
             return this.fetchRetroTemplate(testResults.length)
             .then((template: { workbook: Excel.Workbook, reportTemplate: any} ) => {
+
+                const worksheet = template.workbook.getWorksheet(1);
+                if (testResults.length > 11) {
+                    this.adjustStaticTemplateForMoreThan11Tests(template, testResults);
```

Comment:
Why not only pass testResults length instead of the object as i see you are referring to only the length in these methods.

Reply:
good catch. thanks. fixed

Reply:
Fixed in the function, but doesn't look fixed here?